[18.0][MIG] sale_commercial_partner: Migration to 18.0

LucasTran380381 opened the pull request:
ticket: `https://camptocamp.atlassian.net/browse/BSCOS-5062`

base on: `https://github.com/OCA/sale-workflow/pull/3415`

alexis-via committed:
Initial check-in of module sale_order_import, sale_stock_order_import, sale_order_import_ubl, sale_order_import_csv, sale_order_ubl, sale_commercial_partner

alexis-via committed:
Handle Commercial Customer on sale.report

alexis-via committed:
Port sale_commercial_partner to Odoo 10

OCA Transbot updated translations from Transifex

OCA Transbot updated translations from Transifex

OCA Transbot updated translations from Transifex

OCA Transbot updated translations from Transifex

OCA Transbot updated translations from Transifex

Swapnesh-SerpentCS committed:
[MIG] sale_commercial_partner: Migration to 11.0

[UPD] Update sale_commercial_partner.pot

Swapnesh-SerpentCS committed:
[MIG] sale_commercial_partner: Migration to 12.0

[UPD] README.rst

[UPD] Update sale_commercial_partner.pot

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_commercial_partner/

[UPD] README.rst

liweijie0812 committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_commercial_partner/zh_CN/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_commercial_partner/pt_BR/

kongrattapong committed:
[IMP] sale_commercial_partner: black, isort

kongrattapong committed:
[MIG] sale_commercial_partner: Migration to 13.0

[UPD] Update sale_commercial_partner.pot

[UPD] README.rst

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-13.0/sale-workflow-13.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-13-0/sale-workflow-13-0-sale_commercial_partner/

danimv5 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-13.0/sale-workflow-13.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-13-0/sale-workflow-13-0-sale_commercial_partner/es/

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-13.0/sale-workflow-13.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-13-0/sale-workflow-13-0-sale_commercial_partner/

newtratip committed:
[MIG] sale_commercial_partner: Migration to 14.0

[UPD] Update sale_commercial_partner.pot

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_commercial_partner/nl/

danimv5 committed:
Added translation using Weblate (Catalan)

Translated using Weblate (Catalan)

Currently translated at 100.0% (5 of 5 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_commercial_partner/ca/

Yvesldff committed:
Added translation using Weblate (French (France))

Translated using Weblate (French (France))

Currently translated at 40.0% (2 of 5 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_commercial_partner/fr_FR/

Translated using Weblate (French)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_commercial_partner/fr/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_commercial_partner/de/

santostelmo committed:
[MIG] sale_commercial_partner: Migration to 15.0

alan196 committed:
Added translation using Weblate (Spanish (Mexico))

Translated using Weblate (Spanish (Mexico))

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_commercial_partner/es_MX/

alexis-via committed:
[MIG] sale_commercial_partner to v16

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_commercial_partner/it/

Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_commercial_partner/it/

Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-17.0/sale-workflow-17.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-17-0/sale-workflow-17-0-sale_commercial_partner/it/

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_commercial_partner/hr/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 50.0% (1 of 2 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_commercial_partner/pt_BR/

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_commercial_partner/pt_BR/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_commercial_partner/es/

pilarvargas-tecnativa committed:
[FIX] sale_commercial_partner: add development_status

TT44347

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_commercial_partner/fr/

Commit:
[IMP] sale_commercial_partner: pre-commit auto fixes

Commit:
[MIG] sale_commercial_partner: Migration to 17.0

BertVGroenendael committed:
[IMP] sale_commercial_partner: pre-commit auto fixes

Co-authored-by: Lukas Tran <nhant@trobz.com>

BertVGroenendael committed:
[MIG] sale_commercial_partner: Migration to 18.0

Co-authored-by: Lukas Tran <nhant@trobz.com>

trisdoan commented on sale_commercial_partner/readme/CREDITS.md:
```diff
@@ -0,0 +1,2 @@
+The migration of this module from 13.0 to 14.0 was financially supported
+by Camptocamp
```
mistake?

LucasTran380381 replied:
fixed

trisdoan commented:
It looks to me there are translation commits that could be reordered and squashed, e.g:
[a80235b](https://github.com/LucasTran380381/sale-workflow/pull/6/commits/a80235b66cfd23d1c3e0fc9a698dfdc52c95a4c1) + [ad48e84](https://github.com/LucasTran380381/sale-workflow/pull/6/commits/ad48e840bcee28edb48838290cdc1473fea3baa6)

[9ea2c91](https://github.com/LucasTran380381/sale-workflow/pull/6/commits/9ea2c918069b40dd8fa2e091001275dd3551d8b6) + [4190790](https://github.com/LucasTran380381/sale-workflow/pull/6/commits/419079070e0766cfd62098247f5819afa5331580)

LucasTran380381 commented:
> It looks to me there are translation commits that could be reordered and squashed, e.g: [a80235b](https://github.com/LucasTran380381/sale-workflow/pull/6/commits/a80235b66cfd23d1c3e0fc9a698dfdc52c95a4c1) + [ad48e84](https://github.com/LucasTran380381/sale-workflow/pull/6/commits/ad48e840bcee28edb48838290cdc1473fea3baa6)
> 
> [9ea2c91](https://github.com/LucasTran380381/sale-workflow/pull/6/commits/9ea2c918069b40dd8fa2e091001275dd3551d8b6) + [4190790](https://github.com/LucasTran380381/sale-workflow/pull/6/commits/419079070e0766cfd62098247f5819afa5331580)

squashed commit

trisdoan commented on sale_commercial_partner/README.rst:
```diff
@@ -70,13 +70,27 @@ Authors
+Other credits
+-------------
+
+The migration of this module from 13.0 to 14.0 was financially supported
+by Camptocamp
```
run pre-commit to impact this file plz

LucasTran380381 replied:
re-runed pre-commit

trisdoan reviewed (changes requested):
while at it, can you check the history again? looks to me there are still some squashable commits

LucasTran380381 commented:
> while at it, can you check the history again? looks to me there are still some squashable commits

@trisdoan I squashed translation commit. Thanks
